Elastiflix/java-favorite-otel-auto/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN mvn clean install
 USER root
 RUN apt-get update && apt-get install -y zip curl
 
-ARG OTEL_AGENT_VERSION=1.28.0
+ARG OTEL_AGENT_VERSION=2.0.0
```

Comment:
[for reviewer] the elastic distro currently relies on this version, thus it allows to make the features and behavior consistent.